src/WhatToCook.WebApp/WhatToCook.WebApp.csproj

```diff
@@ -16,6 +16,11 @@
       <PrivateAssets>all</PrivateAssets>
       <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
     </PackageReference>
+    <PackageReference Include="StyleCop.Analyzers" Version="1.1.118">
```

Comment:
As mentioned in Application project, can be removed

Reply:
removed